src/order/filter/germ.lean

```diff
@@ -0,0 +1,534 @@
+/-
+Copyright (c) 2020 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov, Abhimanyu Pallavi Sudhir
+-/
+import order.filter.basic
+import algebra.pi_instances
+
+/-!
+# Germ of a function at a filter
+
+The germ of a function `f : α → β` at a filter `l : filter α` is the equivalence class of `f`
+with respect to the equivalence relation `eventually_eq l`: `f ≈ g` means `∀ᶠ x in l, f x = g x`.
+
+We define `mk l f` to be the germ of `f` at `l` and `const l c` to be the germ of the constant
+function at `l`.
```

Comment:
I think this module docstring should mention the coercions and lifts, and the general `map` before specializing to algebraic structures.